modules/test_user_dashboard/db/migrate/20210226193901_add_idme_uuid_to_tud_accounts.rb

```diff
@@ -0,0 +1,5 @@
+class AddIdmeUuidToTudAccounts < ActiveRecord::Migration[6.0]
+  def change
+    add_column :test_user_dashboard_tud_accounts, :idme_uuid, :string
```

Comment:
Should this be `:uuid` type instead of `:string`?

Reply:
Good point, changed the type